### PR TITLE
Change the print into logging in approx_aggregate_engine and its respective tests

### PR DESCRIPTION
**Pull request by phantominh**
Change from print to logging in the following files, addressing issue  #118 :

- approx_aggregate_engine.py
- test_aggregation.py
- test_aggregation_gaussian.py
- test_aggregation_gaussian_mk.py
- test_exact_aggregation.py

### CODE REVIEW

File: aidb/engine/approx_aggregate_engine.py [hunk 1]
```diff
@@ -58,6 +59,7 @@ async def execute_aggregate_query(self, query: Query):
       estimator = self._get_estimator(agg_type)
       num_samples = self.get_additional_required_num_samples(query, sample_results, estimator)
       print('num_samples', num_samples)
+      logger.info(f'num_samples: {num_samples}')
```
Comment (ddkang) [inline aidb/engine/approx_aggregate_engine.py:62]: The prints should be replaced, not added to

**Comment by ttt-77** [comment]
Could you also check other files and change them? Such as test_limit_engine, test_full_scan_engine, http_inference_service_setup ...

**Comment by ttt-77** [comment]
please check the error @phantominh

**Comment by phantominh** [comment]
> Could you also check other files and change them? Such as test_limit_engine, test_full_scan_engine, http_inference_service_setup ...
Hi, my bad for the last push, I forgot to run tasti_test.py locally. I'm wondering how would you build the test package for this project. Since I could not import tests.<module>, I have been changing import tests.<module> to import <module> to run locally.

**Comment by ttt-77** [comment]
Do you mean how to run test files? python -m tests.test_aggregation

**Comment by ttt-77** [comment]
for tasti_test, you can just run FAISS, and comment another two vector database

**Comment by phantominh** [comment]
Oh I see the issue now, I'm supposed to run it from the parent directory. I'm able to run the test properly now! Thank you!

**Comment by phantominh** [comment]
Hi @ttt-77 , I updated and ran the tests. For submodules inside test (helper files) I don't use the log setup function since that causes import cycle. Its log will be logged to the file handler of the running test, so that shouldn't be an issue.

**Pull request closed without merging**